### PR TITLE
[bulk] Validations: fixes dupH1Titles

### PR DESCRIPTION
**Pull request by Vette99**
This PR fixes dupicateH1Titles, Html, and broken-link issues.

### CODE REVIEW

File: dynamics-nav-app/LocalFunctionality/Germany/how-to-print-test-reports-for-delivery-reminders.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
     ms.author: edupont
 
 ---
-# How to: Print Test Reports for Delivery Reminders
+# How to: Print Test Reports for Delivery Reminders Germany
```
Comment (dereklh7) [inline dynamics-nav-app/LocalFunctionality/Germany/how-to-print-test-reports-for-delivery-reminders.md:18]: Add parentheses to be consistent with title.

File: dynamics-nav-app/LocalFunctionality/Germany/how-to-set-up-delivery-reminders.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
     ms.author: edupont
 
 ---
-# How to: Set Up Delivery Reminders
+# How to: Set Up Delivery Reminders
```
Comment (dereklh7) [inline dynamics-nav-app/LocalFunctionality/Germany/how-to-set-up-delivery-reminders.md:18]: Change to "How to Set Up Deliver Reminders (Germany)" to be consistent with added title.

File: dynamics-nav-app/LocalFunctionality/Germany/how-to-print-test-reports-for-delivery-reminders.md (hunk 1 above)
Comment (Vette99) [reply dynamics-nav-app/LocalFunctionality/Germany/how-to-print-test-reports-for-delivery-reminders.md:18]: Done.

File: dynamics-nav-app/LocalFunctionality/Germany/how-to-set-up-delivery-reminders.md (hunk 2 above)
Comment (Vette99) [reply dynamics-nav-app/LocalFunctionality/Germany/how-to-set-up-delivery-reminders.md:18]: Done.

**Comment by Vette99** [comment]
@edupont04
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by edupont04 — APPROVED** [review]

**Pull request merged**